nemo/deploy/service/config.json

```diff
@@ -0,0 +1 @@
+{"triton_service_ip": "0.0.0.0", "triton_service_port": 8000, "triton_request_timeout": 60, "openai_format_response": false}
```

Comment:
@oyilmaz-nvidia this file needs to be deleted. My previous PR added code to generate this config.json only when `args.start_rest_service` was `True`.  Please see [here](https://github.com/NVIDIA/NeMo/blob/r2.0.0rc1/scripts/deploy/nlp/deploy_triton.py#L352). There's a small bug where `args.start_rest_service` is `str` [here](https://github.com/NVIDIA/NeMo/blob/r2.0.0rc1/scripts/deploy/nlp/deploy_triton.py#L183) and because of that your code is running [store_args_to_json method](https://github.com/NVIDIA/NeMo/blob/r2.0.0rc1/scripts/deploy/nlp/deploy_triton.py#L352). [PR 9834](https://github.com/NVIDIA/NeMo/pull/9834/files#diff-69d857a392e72c8cca395a8738e0ae9fbbd17dbea047c96370d1d1126a2cb63dR183) has the fix to make `args.start_rest_service` `bool`.